src/Chapter1.hs

```diff
@@ -641,7 +654,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit n = if abs n < 10 then abs n else divisible n 
```

Comment:
You can pass `abs n` to `divisible` so you won't need to use `abs` inside 🙂 

You can see that `abs n` is reused several times in this function already. This usually means that you can move it into the `where` case and reuse everywhere 👍🏼

Reply:
Thanks! for the comment 👍